kornia/geometry/calibration/undistort.py

```diff
@@ -21,7 +24,8 @@ def undistort_points(points: torch.Tensor, K: torch.Tensor, dist: torch.Tensor)
         dist: Distortion coefficients
             :math:`(k_1,k_2,p_1,p_2[,k_3[,k_4,k_5,k_6[,s_1,s_2,s_3,s_4[,\tau_x,\tau_y]]]])`. This is
             a vector with 4, 5, 8, 12 or 14 elements with shape :math:`(*, n)`.
-
+        new_K: New intrinsic camera matrix with shape :math:`(*, 3, 3)`. Default: None, in this case K is used.
+        iters: Number of undistortion iterations. Default: 5.
```

Comment:
```suggestion
        num_iters: Number of undistortion iterations. Default: 5.
```
Same for the other

Reply:
done here. What do you refer with "others"?